llm-fine-tuning/notebooks/run_torchdistributor_fine_tuning.py

```diff
@@ -81,6 +81,8 @@
 get_dbutils().widgets.combobox("local_mode", "true", ["true", "false"], "local_mode")
 get_dbutils().widgets.combobox("use_fsdp", "false", ["true", "false"], "use_fsdp")
 get_dbutils().widgets.text("num_gpus", "4", "num_gpus")
+
+# Update the following widget to point to a UC Volumes location if DBFS is not available
```

Comment:
instead of putting the comment which is not "enforcable", can you do the code somethink like 
```
if workspace_is_uc_enabled():
  output_location =  get_dbutils().widgets.text("uc_output_location")
else:
  output_location =  get_dbutils().widgets.text("dbfs_output_location", "/dbfs/llm/", "dbfs_output_location")


....
// before you want to use the output_location
if workspace_is_uc_enabled():
  // make sure to have clear error message here
  verify_is_uc_volumn(output_location)

...
```

Reply:
The main concern is how people access these notebooks. If they are importing the entire git repo as a Databricks repo, then we can provide utility functions like `workspace_is_uc_enabled` since those will still be importable. The question is whether some customers just import singular notebooks into their Databricks workspace, because in those cases, comments would be better than overpopulating a file with unnecessary imports or even worse, unnecessary code implementations of `workspace_is_uc_enabled`

Reply:
@ygong1, talked with @lu-wang-dl during standup and for utility functions like the above, we can create a separate ticket once the UC Volumes team finishes up this ticket: https://databricks.atlassian.net/browse/SC-149810

Reply:
sgtm :)